Microsoft.CSharp.sln fails in msbuild, because of duplicate project name

Microsoft.CSharp.sln fails to build with msbuild (v14.0.24720.0) with following error:

> Solution file error MSB5004: The solution file has two projects named "Microsoft_CSharp".

On the other hand, Visual Studio builds the solution successfully.

Looking at the solution file, the error seems to be that both Microsoft.CSharp.csproj and Microsoft.CSharp.Tests.csproj are named `"Microsoft.CSharp"` in the solution file.

I will submit a pull request to fix the name shortly, but I also have a question: How come automated testing didn't reveal this issue?

--- a/src/Microsoft.CSharp/Microsoft.CSharp.sln
+++ b/src/Microsoft.CSharp/Microsoft.CSharp.sln
@@ -5,7 +5,7 @@ VisualStudioVersion = 12.0.31101.0
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.CSharp", "src\Microsoft.CSharp.csproj", "{96AA2060-C846-4E56-9509-E8CB9C114C8F}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.CSharp", "tests\Microsoft.CSharp.Tests.csproj", "{82B54697-0251-47A1-8546-FC507D0F3B08}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.CSharp.Tests", "tests\Microsoft.CSharp.Tests.csproj", "{82B54697-0251-47A1-8546-FC507D0F3B08}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".nuget", ".nuget", "{A595FB46-5F47-4E70-BDCD-53350879188C}"
 	ProjectSection(SolutionItems) = preProject

